### DISCUSSION TITLE
Test Suite for Code Base

### DISCUSSION THREAD

**Discussion by DuckPaddle** [danielmiessler/Fabric#94] [Ideas]
Is there a Test Suite for the code base?

**Comment by danielmiessler** [comment]
Not yet, we're looking to add tests very soon.

**Comment by donbr** [comment]
@danielmiessler - great concept, and like how you're following some old but good patterns.  (similar to langchain you're using the pipe concept)

I liked the current discussion, and thought the following simple code from Eric Hartford may be a starting point: 
 https://github.com/cognitivecomputations/github2file/blob/main/github2file.py
